certificate/views.py

```diff
@@ -1124,7 +1124,12 @@ def osdag_workshop_download(request):
                 return render_to_response('osdag_workshop_download.html',
                         context, context_instance=ci)
             else:
-                user = user[0]
+                if len(user) > 1:
+                    user_list = [(each_user, int(each_user.year)) for each_user in user]
+                    sorted(user_list, key=lambda x: x[1], reverse=True)
```

Comment:
Instead of this complex sorting, why can't we have django filter ordered by year and give the latest user.

Reply:
removed it. added start and end date field